### PR TITLE
Update ID structure for sea travel

### PR DESCRIPTION
**Pull request by greenercode**
**Is this a new addition or an update to an existing factor?**

- [ ] New
- [x] Update

**What category or categories does it affect?**

Sea travel

What?

For all EFs incurrently in the sea travel category:

IDs updated to adhere to the following machine-readable structure:
passenger_ferry-route_type_[e.g. "foot" or "car"]-route_location_[e.g. "St Malo_to_Portsmouth"] fuel_source_[e.g. "diesel" or "unknown"]-activity_[e.g. "fuel_combustion", "upstream" or "fuel_combustion_and_upstream" or "unknown"]

Name field to be concise, easily read plain English which is as consistent as possible across EFs regardless of source
Description field to provide further detail regarding the specifics of the EF, including life-cycle activity details and specifics of applicability
Updates description field

Why?

This is part of a larger refactoring of IDs across the OEFDB to ensure EFs adhere to a clear standard structure per category and across sources, allowing them to be better understood and compared both by humans and machines.

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -448,9 +448,9 @@ Transport,Vehicle,entry-range-light-hybrid-rechargeable,Entry-Range Light Hybrid
...
-Transport,Sea Travel,ferry-car-passenger,Ferry Car Passenger,kg-CO2e/pkm,0.12952,,BEIS,2021,GB,2021/07/23,Emission Intensity of Ferry Car Passenger. Available in DEFRA's Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
-Transport,Sea Travel,ferry-foot-passenger,Ferry Foot Passenger,kg-CO2e/pkm,0.01874,,BEIS,2021,GB,2021/07/23,Emission Intensity of Ferry Foot Passenger. Available in DEFRA's Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
+Transport,Sea Travel,passenger_ferry-route_type_all_routes_average-route_location_unknown-fuel_source_unknown-activity_fuel_combustion,Average passenger ferry route including fuel combustion,kg-CO2e/pkm,0.11286,,BEIS,2021,GB,2021/07/23,"Emission intensity for an average passenger ferry route (including combustion). Published by BEIS (Department for Business, Energy & Industrial Strategy) in June 2021. ",https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:451]: Use "route_type_unknown" for "all" or "average" EFs

**Review by patstirling — CHANGES_REQUESTED** [review]
- We can remove "route_location_" from these IDs as per other travel categories - the region field does this job.
- the name field should not refer to activity - this belongs in the ID and description fields
- that description field should specify whether upstream emissions are included (they are not for BEIS)

**Review by AnastasiaLobanova — DISMISSED** [review]
Looks good!! just three small typos! :)

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -448,9 +448,9 @@ Transport,Vehicle,entry-range-light-hybrid-rechargeable,Entry-Range Light Hybrid
...
+Transport,Sea Travel,passenger_ferry-route_type_unknown-route_location_unknown-fuel_source_unknown-activity_fuel_combustion,Average passenger ferry route including fuel combustion,kg-CO2e/pkm,0.11286,,BEIS,2021,GB,2021/07/23,"Emission intensity for an average passenger ferry route (including combustion). Published by BEIS (Department for Business, Energy & Industrial Strategy) in June 2021. ",https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
+Transport,Sea Travel,passenger_ferry-route_type_car-route_location_unknown-fuel_source_unknown-activity_fuel_combustion,Ferry route including including cars and passengers including fuel combustion,kg-CO2e/pkm,0.12952,,BEIS,2021,GB,2021/07/23,"Emission Intensity for a ferry route including including cars and passengers (including combustion). Published by BEIS (Department for Business, Energy & Industrial Strategy) in June 2021. ",https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:452]: Please do not include activity in the name.

[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:452]: Do we need " in the description?

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -448,9 +448,9 @@ Transport,Vehicle,entry-range-light-hybrid-rechargeable,Entry-Range Light Hybrid
...
-Transport,Sea Travel,ferry-car-passenger,Ferry Car Passenger,kg-CO2e/pkm,0.12952,,BEIS,2021,GB,2021/07/23,Emission Intensity of Ferry Car Passenger. Available in DEFRA's Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
-Transport,Sea Travel,ferry-foot-passenger,Ferry Foot Passenger,kg-CO2e/pkm,0.01874,,BEIS,2021,GB,2021/07/23,Emission Intensity of Ferry Foot Passenger. Available in DEFRA's Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
+Transport,Sea Travel,passenger_ferry-route_type_unknown-route_location_unknown-fuel_source_unknown-activity_fuel_combustion,Average passenger ferry route including fuel combustion,kg-CO2e/pkm,0.11286,,BEIS,2021,GB,2021/07/23,"Emission intensity for an average passenger ferry route (including combustion). Published by BEIS (Department for Business, Energy & Industrial Strategy) in June 2021. ",https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:451]: Do you know what average refers to here?

File: OpenEmissionFactorsDB.csv (hunk 2 above)
[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:452]: Is "car" a route type?

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -448,9 +448,9 @@ Transport,Vehicle,entry-range-light-hybrid-rechargeable,Entry-Range Light Hybrid
...
+Transport,Sea Travel,passenger_ferry-route_type_car-route_location_unknown-fuel_source_unknown-activity_fuel_combustion,Ferry route including including cars and passengers including fuel combustion,kg-CO2e/pkm,0.12952,,BEIS,2021,GB,2021/07/23,"Emission Intensity for a ferry route including including cars and passengers (including combustion). Published by BEIS (Department for Business, Energy & Industrial Strategy) in June 2021. ",https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
+Transport,Sea Travel,passenger_ferry-route_type_foot-route_location_unknown-fuel_source_unknown-activity_fuel_combustion,Ferry route with foot passengers only including fuel combustion,kg-CO2e/pkm,0.01874,,BEIS,2021,GB,2021/07/23,"Emission Intensity for ferry route with foot passengers only (including combustion). Published by BEIS (Department for Business, Energy & Industrial Strategy) in June 2021. ",https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:453]: Please do not include activity in the name.

Comment (greenercode) [reply OpenEmissionFactorsDB.csv:453]: Ok!

File: OpenEmissionFactorsDB.csv (hunk 3 above)
Comment (greenercode) [reply OpenEmissionFactorsDB.csv]: Yes, average (all passenger).

**Comment by patstirling** [comment]
> * We can remove "route_location_" from these IDs as per other travel categories - the region field does this job.
> * There appear to have been quotation marks added to each of the descriptions - could these be removed
> * "Emission Intensity" -> "Emission intensity" in all description fields

**Review by patstirling — CHANGES_REQUESTED** [review]
See earlier comments

**Comment by patstirling** [comment]
Also I see 449-450 and 455 have been altered; please revert

**Pull request closed without merging**